src/Illuminate/Validation/Validator.php

```diff
@@ -1235,7 +1235,9 @@ protected function validateUrl($attribute, $value)
      */
     protected function validateActiveUrl($attribute, $value)
     {
-        $url = str_replace(['http://', 'https://', 'ftp://'], '', strtolower($value));
```

Comment:
The new logic simply ignores all protocols and only extracts the hostname from the given value.  So, even adding test cases with `ftp://` and `https://` protocols isn't going to really test anything, other than that PHP's internal `parse_url()` function works.